exercises/error-handling/example.rs

```diff
@@ -0,0 +1,81 @@
+use std::cmp;
+
+#[derive(Eq,PartialEq,Debug)]
+pub enum IceCream {
+    Chocolate,
+    Vanilla,
+    Strawberry,
+}
+
+impl IceCream {
+    /// Convert various string versions of ice cream flavors to a
+    /// proper IceCream enum or return Error
+    pub fn get_flavor(request: &str) -> Result<IceCream, &'static str> {
```

Comment:
makes me wish https://doc.rust-lang.org/std/convert/trait.TryFrom.html were stable so we could use it here.